app/frontend/src/components/TOS.tsx

```diff
@@ -0,0 +1,34 @@
+import { Link } from "@material-ui/core";
+import React from "react";
+import PageTitle from "./PageTitle";
+import BodyText from "./TextBody";
+
+export default function TOS() {
+  return (
+    <>
+      <PageTitle>Terms of Service</PageTitle>
+      <BodyText>
+        This is a test TOS. I agree to follow the&nbsp;
+        <Link href="https://community.couchers.org/faq">
+          Couchers guidelines
+        </Link>
+        . I understand this is a preview of couchers.org and my data may be
+        erased. Lorem Ipsum is simply dummy text of the printing and typesetting
+        industry.
+        <br />
```

Comment:
I hope the final text will use `<TextBody>`.

Reply:
Whoops, how did I manage to do that??